app/helpers/articles_helper.rb

```diff
@@ -56,4 +56,8 @@ def get_host_without_www(url)
   def utc_iso_timestamp(timestamp)
     timestamp&.utc&.iso8601
   end
+
+  def active_threads(**options)
```

Comment:
Ruby 2.7 added argument forwarding, so theoretically we could change this method:

```ruby
def active_threads(...)
  Articles::ActiveThreadsQuery.call(...)
end
```

Given that we don't use this syntax elsewhere we probably shouldn't, but it's good to be aware of the possibility.

Reply:
Thanks for bringing it to my attention--I'll keep it in mind! 😊